views/layouts/default.lavender

```diff
@@ -22,11 +22,14 @@ head
   meta(name='msapplication-TileColor', content='#cdb4cc')
   meta(name='msapplication-TileImage', content='/img/favicons/mstile-144x144.png')
   meta(name='theme-color', content='#796878')
-  link(rel='stylesheet', href='http://fonts.googleapis.com/css?family=Oswald:400,600|Lato:300,700|Special+Elite')
+  link(rel='stylesheet',
+    href='http://fonts.googleapis.com/css?family=Oswald:400,600|Lato:300,700|Special+Elite'
+  )
   link(rel='stylesheet', 
     href='http://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css')
```

Comment:
Try to keep the formatting consistent. Put the trailing `)` on a new line. Also, when breaking these paren attributes into multiple lines, I generally put the first one on a new line as well. I'm pretty sure this is standard.

```
link(
  rel='stylesheet',
  href='blablabla'
)